sdk/eventhub/event-hubs/test/sender.spec.ts

```diff
@@ -244,21 +244,22 @@ describe("EventHub Sender #RunnableInBrowser", function(): void {
       }
     });
 
-    it("should fail when a message greater than 256 KB is sent and succeed when a normal message is sent after that on the same link.", async function(): Promise<
+    it("should fail when a message greater than 1 MB is sent and succeed when a normal message is sent after that on the same link.", async function(): Promise<
```

Comment:
This test was previously returning a false positive, because it was not throwing an error. At some point Event Hubs raised the message limit to 1 MB:
https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-quotas

I wonder if it's worth having this test since we're actually testing the service and not the SDK.

Reply:
I believe what is actually being tested here is that a failure to send a message should not stop the sender link from sending subsequent messages. So, unless we have a way to induce failure on the link, we can keep this